documentation/asciidoc/computers/raspberry-pi/introduction.adoc

```diff
@@ -27,12 +27,12 @@ a|
 512MB | 26-pin GPIO header
 a|
 * HDMI
-* 2 × USB 2.0
+* 2× USB 2.0
 * CSI camera port
 * DSI display port
 * 3.5mm audio jack
 * RCA composite video
-* Ethernet (100Mb/s)
+* 100Mb/s Ethernet RJ45 with PoE support
```

Comment:
No PoE on the models 1B, 2B or 3B! It got added in 3B+, see https://www.raspberrypi.com/news/raspberry-pi-3-model-bplus-sale-now-35/ and https://www.raspberrypi.com/news/introducing-power-over-ethernet-poe-hat/